app/src/main/java/com/github/se/icebreakrr/ui/profile/InboxProfileView.kt

```diff
@@ -0,0 +1,241 @@
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavBackStackEntry
+import com.github.se.icebreakrr.model.message.MeetingRequestViewModel
+import com.github.se.icebreakrr.model.profile.ProfilesViewModel
+import com.github.se.icebreakrr.model.tags.TagsViewModel
+import com.github.se.icebreakrr.ui.navigation.NavigationActions
+import com.github.se.icebreakrr.ui.sections.shared.InfoSection
+import com.github.se.icebreakrr.ui.sections.shared.MessageWhenLoadingProfile
+import com.github.se.icebreakrr.ui.sections.shared.ProfileHeader
+
+private const val ACCEPT_DECLINE_ELEVATION = 4
+private const val ACCEPT_DECLINE_ROUNDER_CORNER_SHAPE = 23
+private const val ACCEPT_DECLINE_SHADOW_COLOR = 0x40000000
+private const val ACCEPT_DECLINE_BOX_COLOR = 0xFFEAEEFF
+private const val ACCEPT_DECLINE_BOX_HEIGHT = 115
+private const val ACCEPT_DECLINE_BOX_WIDTH_FACTOR = 0.9
+private const val ACCEPT_DECLINE_PADDING_START = 10.5
+private const val ACCEPT_DECLINE_PADDING_END = 14.5
+private const val ACCEPT_DECLINE_PADDING_TOP_BOTTOM = 8
+private const val REQUEST_MESSAGE_FONT_SIZE = 20
+private const val REQUEST_MESSAGE_LINE_HEIGHT = 20
+private const val REQUEST_MESSAGE_FONT_WEIGHT = 500
+private const val REQUEST_MESSAGE_LETTER_SPACING = 0.2
+private const val SPACER_HEIGHT = 10
+private const val ACCEPT_DECLINE_ICON_BUTTON_PADDING = 12
+private const val ACCEPT_DECLINE_ICON_BUTTON_SIZE = 48
+private const val ACCEPT_DECLINE_ICON_BUTTON_ROUNDED = 100
+private const val ACCEPT_DECLINE_ICON_BUTTON_COLOR = 0xFF65558F
+
+/**
+ * Screen that appears when you click on a profile in the notification tab. You can accept or
+ * decline the notification.
+ *
+ * @param profilesViewModel : view model of the profile
+ * @param navBackStackEntry : used to retrieve the uid of the user that we clicked on
+ * @param navigationActions : go back when we accept, decline or go back
+ * @param tagsViewModel : used to display user's tag
+ * @param meetingRequestViewModel : view model to send/receive meeting requests
+ * @param isTesting : indicates if we are in testing mode or not
+ */
+@Composable
+fun InboxProfileViewScreen(
+    profilesViewModel: ProfilesViewModel,
+    navBackStackEntry: NavBackStackEntry?,
+    navigationActions: NavigationActions,
+    tagsViewModel: TagsViewModel,
+    meetingRequestViewModel: MeetingRequestViewModel,
+    isTesting: Boolean
+) {
+
+  LaunchedEffect(Unit) {
```

Comment:
Improvement: Simplify if (profileId != null || isTesting) to avoid redundant null checks 

val profileId = navBackStackEntry?.arguments?.getString("userId") ?: ""
if (isTesting || profileId.isNotEmpty()) {
    profilesViewModel.getProfileByUid(profileId)
    meetingRequestViewModel.updateInboxOfMessages()
}

Reply:
This doesn't really work because If I want to do what you said I need to do  : 
```
if (isTesting || profileId?.isNotEmpty() == true) {
    profilesViewModel.getProfileByUid(profileId?:"")
    meetingRequestViewModel.updateInboxOfMessages()
}
```
I still need to check for null values in `profileId` and `profileId.isNotEmpty()` cannot be called like that because `profileId `can be null